roles/inspec/templates/etc/inspec/host-controls/attributes.yml

```diff
@@ -1,3 +1,6 @@
 {% for name,value in inspec.openstack.horizon.attributes.iteritems() %}
 {{ name }}: {{ value }}
 {% endfor %}
+{% for name,value in inspec.rhel7.attributes.iteritems() %}
```

Comment:
Currently we do not have support for Ubuntu Inspec checks, but we will add the attributes for it when we have that in place.